pkgs/development/tools/continuous-integration/jenkins-job-builder/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, pythonPackages, buildPythonPackage, git }:
+
+buildPythonPackage rec {
+  name = "1.2.0";
```

Comment:
@jagajaga: I think that makes the final package name ```jenkins-job-builder-jenkins-job-builder``` (double name), right? I would suggest ```namePrefix = ""``` and ```name = "jenkins-job-builder-${version}"```.